learnosity-sdk.gemspec

```diff
@@ -32,8 +32,8 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency 'sys-uname', '>= 1.0'
 
-  spec.add_development_dependency 'bundler', '~> 1.14'
-  spec.add_development_dependency 'rake', '~> 10.0'
+  spec.add_development_dependency 'bundler', '~> 2.2.10'
```

Comment:
Instead of specifying this version, which is only available by default for Ruby 3.0.2 installations, could we use `'>= 1.17.2'`? This covers all the versions of Ruby we test with.